lib/src/layer/polygon_layer/label.dart

```diff
@@ -85,7 +85,7 @@ LatLng _computePolylabel(List<LatLng> points) {
     // point with more distance to the polygon's outline. It's given in
     // point-units, i.e. degrees here. A bigger number means less precision,
     // i.e. cheaper at the expense off less optimal label placement.
-    precision: 0.000001,
+    precision: 0.0001,
```

Comment:
Based on [this](https://wiki.openstreetmap.org/wiki/Precision_of_coordinates) 4 decimal places can alter up to about 11 meters on the equator. Are we sure that we want to have this change?

If we lower the percision we should at least give users the possibility to configure this value.

Reply:
With the current precision, it sometimes caused lag so bad that it literally crashed the debug engine and app.
We should ideally make it an option, but that's not for this PR. I'll add a TODO.